app/views/finders/_show_header.html.erb

```diff
@@ -32,6 +32,15 @@
             value: result_set_presenter.user_supplied_keywords,
           } %>
         </div>
+        <div id="js-spelling-suggestions" class="spelling-suggestions" 
+          data-analytics-ecommerce
+          data-ecommerce-start-index="<%= result_set_presenter.start_offset %>"
```

Comment:
Is it useful to track what page the user is on?  Spelling suggestions are a separate list to search results

Reply:
do you mean it's missing and we need to, or it's there but it's redundant?

Reply:
I think the `data-ecommerce-start-index` is redundant for spelling suggestions

Reply:
that determines the position in the list, if i take it out, it says 
Impression List 1 Product 1 Position : NaN
I can just say 1 - as it's just one for now

Reply:
I think just saying `1` would be good, as `start_offset` will be change if the user goes to a different page, but the page they're on doesn't have any bearing on the spelling suggestion they get.

Reply:
cool, updated